.circleci/config.yml

```diff
@@ -942,26 +942,6 @@ jobs:
 workflows:
   build-test:
     jobs:
-    - build-binaries:
```

Comment:
Is this just a temporary removal for purposes of testing-out this PR more quickly?

Reply:
🤦‍♂ Oops - that was only meant for the branch I checking for flakiness